### PR TITLE
TN-1163 removed u in intervention_strategies.py

### PR DESCRIPTION
**Pull request by John0Randolph**

### CODE REVIEW

File: portal/models/intervention_strategies.py [hunk 1]
```diff
@@ -231,15 +233,15 @@ def update_user_card_html(intervention, user):
                 classification='indefinite'))
 
         def thank_you_block(name, registry):
-            greeting = _(u"Thank you, {}.").format(name)
+            greeting = _("Thank you, {}.").format(name)
```
Comment (ivan-c) [inline portal/models/intervention_strategies.py]: Can you change this to use printf-like syntax? FYI The variable name (`name`) gives translators extra info about how to translate the message:

```
_("Thank you, %(full_name)s.", full_name=name)
```

File: portal/models/intervention_strategies.py [hunk 2]
```diff
@@ -263,7 +265,7 @@ def intro_html(assessment_status):
 
             if assessment_status.overall_status in (
                     'Due', 'Overdue', 'In Progress'):
-                greeting = _(u"Hi {}").format(user.display_name)
+                greeting = _("Hi {}").format(user.display_name)
```
Comment (ivan-c) [inline portal/models/intervention_strategies.py]: Same as above:
```
_("Hi, %(full_name)s.", full_name=display_name)

```

File: portal/models/intervention_strategies.py (hunk 1 above)
Comment (John0Randolph) [reply portal/models/intervention_strategies.py]: On it

File: portal/models/intervention_strategies.py [hunk 3]
```diff
@@ -263,7 +265,7 @@ def intro_html(assessment_status):
 
             if assessment_status.overall_status in (
                     'Due', 'Overdue', 'In Progress'):
-                greeting = _(u"Hi {}").format(user.display_name)
+                greeting = _("Hi, %(full_name)s", full_name=user.display_name)
```
Comment (ivan-c) [inline portal/models/intervention_strategies.py]: Extra comma :)

File: portal/models/intervention_strategies.py [hunk 4]
```diff
@@ -301,12 +303,12 @@ def intro_html(assessment_status):
                     </div>""".format(greeting=greeting, reminder=reminder)
 
             if any(indefinite_questionnaires):
-                greeting = _(u"Hi {}").format(user.display_name)
+                greeting = _("Hi, %(full_name)s", full_name=user.display_name)
```
Comment (ivan-c) [inline portal/models/intervention_strategies.py]: Extra comma :)

**Pull request merged**